docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,174 @@
+
+These APIs are used to manage Momento authentication tokens and access. Tokens can be scoped to have one or more roles to give access to one or more caches or topics.
+
+## GenerateAuthToken
+
+Generates a new Momento authentication token with the specified permissions and expiry.
+
+| Name            | Type                       | Description                                                                                       |
+| --------------- |----------------------------|---------------------------------------------------------------------------------------------------|
+| scope           | [TokenScope](#tokenscope)  | The permissions to grant to the new token. Pre-built TokenScope objects are provided by the SDKs. |
+| expiresIn       | Number \| ExpiresIn object | The number of seconds until the token expires or an ExpireIn object calling the `ExpireIn.never()` method.  |
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+  - `authToken`: string - the new auth token
+  - `refreshToken`: string - a refresh token that can be used with the [RefreshAuthToken API](#refreshauthtoken) to refresh a token before it expires
+  - `expiresAt`: Timestamp - the timestamp at which the token will expire
+* Error
+
+See [response objects](./response-objects) for specific information.
+
+</details>
+
+:::note
+
+Tokens to access the Momento control plane APIs can only be generated using the [Momento console](https://console.gomomento.com/).
+
+:::
+
+<SdkExampleTabs snippetId={'API_GenerateAuthToken'} />
+
+## RefreshAuthToken
+
+Refreshes an existing, unexpired Momento auth token.  Produces a new auth token with the same permissions and expiry duration as the original auth token.
+
+| Name            | Type            | Description                                   |
+| --------------- | --------------- | --------------------------------------------- |
+| refreshToken    | String          | The refreshToken for the current auth token, acquired from the original call to `GenerateAuthToken`. |
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+  - `authToken`: string - the new auth token
+  - `refreshToken`: string - a refresh token that can be used with the [RefreshAuthToken API](#refreshauthtoken) to refresh the token before it expires
+  - `expiresAt`: Timestamp - the timestamp at which the token will expire
+* Error
+
+See [response objects](./response-objects) for specific information.
+
+</details>
+
+<SdkExampleTabs snippetId={'API_RefreshAuthToken'} />
+
+<img src="/img/momento-auth-tokens.png" />
+
+## TokenScope
+| Name            | Type                                       | Description                                  |
+| --------------- |--------------------------------------------| -------------------------------------------- |
+| permissions           | Array <[Permission](#permission-objects)\> | The permissions to grant to the new token.|
```

Comment:
I think I used `Array` in my draft too but List might be a more generic term.

Reply:
Fixed.